### DISCUSSION TITLE
Licensing Questions

### DISCUSSION THREAD

**Discussion by suhoKKK** [BinaryKits/ZPLUtility#168] [Q&A] [answered]
Hello, I am developing a C# program at my company. I want to use this package because I need to implement the preview features of ZEBRA ZPL in my project. I wonder if I can use this package and if I use the package's own viewer it won't collect data or create security issues differently than Labelary. Also, I wonder if there is no limit on the number of preview views, etc. thank you

**Comment by tinohager** [comment, marked as answer]
There is no limit to preview images and it also does not require an external connection
